### ISSUE TITLE
Why do you comment out codes which matches Algorithm 1 ?

### ISSUE THREAD

**Issue by yw981** [jfc43/robust-ood-detection#2]
confidence_pgd_attack.py line 92~102

           ` # one_hot_labels = torch.eye(len(outputs[0]))[y].to(CUDA_DEVICE)
            # other, _ = torch.max((1-one_hot_labels)*outputs, dim=1)
            # correct = torch.masked_select(outputs, one_hot_labels.byte())
            # loss = torch.clamp(other - correct, min=-50.0)
            # if self.in_distribution:
                # outputs = F.log_softmax(outputs, dim=1)
                # uniform_dist = torch.Tensor(x.size(0), self.num_classes).fill_((1./self.num_classes)).cuda()
                # loss = self.loss_func(outputs, uniform_dist)

            # else:
                # loss = self.loss_func(outputs)`

which matches Algorithm 1 in your paper.

Why do you comment the codes out?
